lib/utils.h

```diff
@@ -78,4 +78,7 @@ inline static const char *getOrdinalText(int i)
     return "th";
 }
 
+template<typename T>
+inline void UNUSED(T&&) {}
```

Comment:
Does this really require such a template? (void)param; does the same job, doesn't it?